internal/pkg/cli/validate.go

```diff
@@ -60,6 +60,11 @@ var (
 	fmtErrInvalidEngineType        = "invalid engine type %s: must be one of %s"
 	fmtErrInvalidDBNameCharacters  = "invalid database name %s: must contain only alphanumeric characters and underscore; should start with a letter"
 	errInvalidSecretNameCharacters = errors.New("value must contain only letters, numbers, periods, hyphens and underscores")
+
+	// Topic subscription errors.
+	errMissingPublishTopicField = errors.New("field `publish.topics[].name` cannot be empty")
+	errInvalidPubSubTopicName   = errors.New("topic names can only contain letters, numbers, underscores, and hypthens")
```

Comment:
classic @kohidave mistake:
```suggestion
	errInvalidPubSubTopicName   = errors.New("topic names can only contain letters, numbers, underscores, and hyphens")
```